src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
```

Comment:
Add input validation: The code assumes that the people array always contains valid objects with the required properties. It's better to add input validation to ensure that the function doesn't break when given unexpected input. For example: 
const people = people.includes(obj, 0);